### ISSUE TITLE
Build and style header

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#58]
# Done criteria
- [x] Build and style header
- [x] Contains the nav options [shown here](https://invis.io/3PEAE33G4)
- [x] **ISSUE** they'd like a FB icon at the top (Cat is designing it)
- [x] **ISSUE** they'd like a bit more vertical space between the logo and the nav (cat is designing it)
- [x] **ISSUE FOR DEMO** Make sure join button is white
- [x] **ISSUE FOR DEMO** Make sure Google+  button doesn't appear in the header

# Designs
Design on [our Purpose page](https://projects.invisionapp.com/share/3PEAE33G4#/screens)

# BGS review criteria
[Link to review](http://bgs-dev.mrmdev.co.uk/about)
- [x] Looks and work like design

**Comment by angievanhegan** [comment]
@pipe-manta I'm moving this into blocked/tricksy as there are a couple of amends from the client and you'll need designs from cat (issue #44)

**Comment by angievanhegan** [comment]
@pipe-manta This is now unblocked - the two amends (shown as ISSUES at the top of here) have now been signed off.

**Comment by ican-icant** [comment]
@pipe-manta I've just noticed the crumbs bar is too high - please see the design and reduce the height of it. It has too much prominence.

**Comment by StephMSlack** [comment]
@ican-icant Hi Steve, screen shots of how the page is looking on my screen. Likely due to elements of the page not being fully styled as yet. Here are the notes I made:

RHS Alignment of the nav bar - should line up with the join/log out button above
RHS social media buttons feel more prominent than in design 
BGS logo - too left aligned (impacts footer, breadcrumb trail (should start below logo, currently doesn't) and amount of grey space border around content box




<img width="1438" alt="screen shot 2017-11-22 at 16 37 58" src="https://user-images.githubusercontent.com/32127102/33138983-dda7efe2-cfa3-11e7-823a-c510725b3084.png">
<img width="1437" alt="screen shot 2017-11-22 at 16 37 45" src="https://user-images.githubusercontent.com/32127102/33138984-ddbbf794-cfa3-11e7-9f99-c503ad39b9ca.png">
<img width="1436" alt="screen shot 2017-11-22 at 16 37 13" src="https://user-images.githubusercontent.com/32127102/33138985-ddd102ba-cfa3-11e7-8063-a43b3ab81131.png">

**Comment by pipe-manta** [comment]
Hi @StephMSlack and @ican-icant, I have checked the following points and looks fine now:

- RHS alignment of the footer menu with the top menu ( join/log out buttons)
- Social menu in the footer was the same as the design
- LHS alignment between the logo, breadcrumbs, footer information and footer menu
- The amount of space around the elements on the additional section has been removed.

All the best,
Pipe

**Comment by angievanhegan** [comment]
I moved these issues to #58 which relates to the footer

**Comment by StephMSlack** [comment]
Hi @cat-manta - could you confirm what the hover colour should be for the masthead & footer? Feedback from a couple of staff here is that the pink is looking too 'girly'. Plus it doesn't feel that strong with the pink on top of the purple.

**Comment by angievanhegan** [comment]
@StephMSlack I've put that specific challenge into a new issue (#81) for @cat-manta to focus on later. Are you ok to sign the rest off? Thanks
